package.json

```diff
@@ -1,23 +1,39 @@
 {
-  "name": "purdue-laundry-api",
+  "name": "purduelaundry-server",
   "version": "1.0.0",
-  "description": "Server for parsing things in my first attempt at NodeJS",
+  "description": "The API Server for ACM SigApp's Purdue Laundry app",
   "main": "index.js",
+  "repository": "https://github.com/purdue-acm-sigapp",
```

Comment:
Needs to point to this repo, not the org I believe.

Reply:
good point. not really sure why i did this lol